src/dashboard/pages/CheckoutPage.vue

```diff
@@ -1,18 +1,83 @@
 <template lang="pug">
 v-container
-  donation-summary(ref='donationSummary', @error="errfun('Donating failed')($event)" checkout="true")
+  v-stepper(:value="step")
+    v-stepper-header
+      v-stepper-step(step="1", :complete="step > 1") Review your donations
+      v-divider
+      v-stepper-step(step="2", :complete="step > 2") Sign transactions
+      v-divider
+      v-stepper-step(step="3", :complete="step > 2") Done
+
+    v-stepper-content(step="1")
+      donation-summary(ref='donationSummary', @error="transactionError", checkout="true", :distribution="distribution")
+      v-layout(row)
+        v-flex
+        v-flex(shrink)
+          v-btn(color="primary", @click="processTransactions()") Continue 
+    v-stepper-content(step="2")
+      h3 Sign transactions in metamask
+      v-list
+        v-list-tile(v-for="d in pendingDonations", :key="d.id")
+          v-list-tile-action
+            v-progress-circular(v-if="d.status === 'pending'", indeterminate, color="primary")
+            v-icon(v-else-if="d.status === 'failed'") error
+            v-icon(v-else-if="d.status === 'completed'", color="primary") check_circle
+          v-list-tile-content
+            v-list-tile-title(v-text="d.name")
+    v-stepper-content(step="3")
+      h3 All done! Now just keep surfin' baby.
```

Comment:
lol, change to slightly less Patrik-y message?

Reply:
Something like 'All donations where successful! You can now go back to browsing as usual. We will remind you when it is time to donate again.' ?

Reply:
That sounds reasonable